loopy/version.py

```diff
@@ -42,7 +42,7 @@
 # }}}
 
 
-VERSION = (2020, 1)
+VERSION = (2020, 2)
 VERSION_STATUS = ""
```

Comment:
Actually, we should discuss how this interacts with language versions. My read on this is that we're essentially dropping support for language versions 2018.1 and prior. They should be rejected at `make_kernel` and their documentation removed. `FALLBACK_KERNEL_VERSION` should be bumped to 2018.2.

Reply:
Yes, I agree. Can't find a less stricter solution, so will push the above mentioned changes.

Reply:
One problem: some kernels for which we generated codes in lang versions `2018.2` would now raise in loopy version `2020.2`. Because when we added lang version `2018.2` [a bug](https://github.com/inducer/loopy/blob/a1e46b6205902f19d541d8abf9f5382ecab9c8ad/loopy/check.py#L790-L791) would silently let them pass through.

Reply:
- Could you confirm that this doesn't apply to anything in Loopy's own test suite, correct?
- Removing this support is consistent with documented behavior, even if the warnings weren't issued. So I'm not super fazed by this.

Reply:
> Could you confirm that this doesn't apply to anything in Loopy's own test suite?

It led to the changes in the tests as a part of this PR.

> Removing this support is consistent with documented behavior, even if the warnings weren't issued. So I'm not super fazed by this.

Yep, I also realized this (a bit later) and hence 68d7217.